web/src/lib/components/FancyHeader.svelte

```diff
@@ -0,0 +1,87 @@
+<script lang="ts">
+	export let headerLevel = '1';
+</script>
+
+<div class="fancy-header">
+	<div class={`icon icon-level-${headerLevel}`}>
+		<slot name="icon"></slot>
+	</div>
+	<svelte:element this={`h${headerLevel}`}>
+		<slot></slot>
+	</svelte:element>
+</div>
+
+<style>
+	.fancy-header {
+		--slant-px: 4rem;
+		background: lightgray;
+		padding: 0.6em 2.5em 0.6em 1em;
+    /* Should hold the negative of the horizontal size of the fold */
+		margin-left: -10px;
```

Comment:
I did a grep for px and there are still some occurrences.

I took a look at the page with the new divider  and I assume that the divider is thicker because if you set it too thin it will disappear. I guess that the best solution is to use `px` to set a lower limit and relative sizing to set the actual size that you want.

I think that at this stage this is probably too much work/fine tuning for the first web pages scaffold 😛, I'm sorry that I ever brought up pixels!

Reply:
Some of em are left pixels precisely cause they're the best for the job actually. They are sizes for the robot, or this little paper fold. Using relative sizes would be a nightmare.

But yeah, we can mull over absolutes vs relatives later on and not right now. They won't make or break designs after all.